scripts/in_container/run_install_and_test_provider_packages.sh

```diff
@@ -38,13 +38,10 @@ if (($# < 1)); then
     exit 2
 fi
 
-INSTALL_TYPE=${1}
-readonly INSTALL_TYPE
-
-if [[ ${INSTALL_TYPE} != "whl" && ${INSTALL_TYPE} != "tar.gz" ]]; then
-    echo
-    echo  "${COLOR_RED_ERROR} Wrong install type ${INSTALL_TYPE}. Should be 'whl' or 'tar.gz'  ${COLOR_RESET}"
-    echo
+if [[ ${PACKAGE_FORMAT} != "wheel" && ${PACKAGE_FORMAT} != "sdist" ]]; then
```

Comment:
This is going to conflict with #12685 which adds `both` :)

Reply:
Yeah. I am going to resolve it.

Reply:
Actually I am going to merge that one after #12685 - exactly because they will fix the CI failures.